Jenkinsfile.yml

```diff
@@ -24,10 +26,21 @@ node('ccode'){
       '''
     }
 
+    stage('Build TinyCBOR'){
+      sh '''
+        cd $WORKSPACE/tinycbor
+        echo 'Building TinyCBOR'
+        git checkout v0.5.3
```

Comment:
Would it be better to use a variable for the version? Or at least, put a comment at the top of this file to indicate that tinycbor and safestring versions should be updated if necessary.

Reply:
Used for CI setup and used only at this one place.